addons/google_calendar/static/src/js/google_calendar.js

```diff
@@ -190,40 +209,35 @@ const GoogleCalendarRenderer = CalendarRenderer.include({
     }),
 
     events: _.extend({}, CalendarRenderer.prototype.events, {
-        'click .o_google_sync_button': '_onGoogleSyncCalendar',
-        'click .o_stop_google_sync_button': '_onStopGoogleSynchronization',
+        'click .o_google_sync_pending': '_onGoogleSyncCalendar',
+        'click .o_google_sync_button_configured': '_onStopGoogleSynchronization',
     }),
 
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
 
     _initGooglePillButton: function() {
-        this.$googleStopButton.css({"cursor":"pointer", "font-size":"0.9em"});
-        var switchBadgeClass = (elem) => {elem.toggleClass('badge-primary'); elem.toggleClass('badge-danger');};
-        this.$('.o_stop_google_sync_button').hover(() => {
-            switchBadgeClass(this.$googleStopButton);
-            this.$googleStopButton.html("<i class='fa mr-2 fa-times'/>".concat(_t("Stop the Synchronization")));
+        // hide the pending button
+        this.$calendarSyncContainer.find('#google_sync_pending').hide();
+        const switchBadgeClass = (elem) => {elem.toggleClass('badge-primary'); elem.toggleClass('badge-danger');};
```

Comment:
I'm not sure but I think we could do this:
```suggestion
        const switchBadgeClass = (elem) => { elem.toggleClass('badge-primary').toggleClass('badge-danger'); };
```

Reply:
It's possible to pass an array to `toggleClass`, so we can do this:

```suggestion
        const switchBadgeClass = elem => elem.toggleClass(['badge-primary', 'badge-danger']);
```